src/htm/regions/RDSERegion.cpp

```diff
@@ -47,18 +48,23 @@ namespace htm {
           resolution:  {type: Real32, default: "0.0"},
           category:    {type: Bool,   default: "false"},
           seed:        {type: UInt32, default: "0"},
-          sensedValue: {type: Real64, default: "0.0", access: ReadWrite }},
+          noise:       {description: "amount of noise to add to the output SDR. 0.01 is 1%",
```

Comment:
nice idea to have the noice included in the encoder :+1: 
Maybe for later, it'd be useful to have a "NoisyRegion", so that it can be applied to/after any layer. That is similar to dropout PR #535

Reply:
Good idea.